src/components/Common/ProfileSelector/SelectButton/SelectButton.tsx

```diff
@@ -46,13 +46,15 @@ export default function SelectButton<T>(props: SelectButtonSpec<T>): JSX.Element
 				onClick={onClick}
 				className="bg-blue-200 hover:bg-yellow-dark-yellow text-gray-800 font-bold py-1 px-1 rounded-full inline-flex items-center transition duration-300"
 			>
-				<svg viewBox="0 0 32 32"
+				<svg 
+				  width="24px"
+					height="24px"
+				  viewBox="0 0 32 32"
```

Comment:
What's up with indentation?

Reply:
I hate 2 tab indentation, it's really ugly :(

Reply:
Idk what you're talking about, but there was simply 2 space on line 50 and 52 that were fucking up the whole thing!